quinn/examples/common/mod.rs

```diff
@@ -0,0 +1,77 @@
+//! Commonly used code in most examples.
+
+use quinn::{
+    Certificate, CertificateChain, ClientConfig, ClientConfigBuilder, Driver, Endpoint, Incoming,
+    PrivateKey, ServerConfig, ServerConfigBuilder, TransportConfig,
+};
+use std::error::Error;
+use std::net::ToSocketAddrs;
+use std::sync::Arc;
+
+/// Returns QUIC endpoint builder configured to build a client endpoint.
+///
+/// ## Args
+///
+/// - server_certs: list of trusted certificates.
+#[allow(unused)]
+pub fn make_client_endpoint<A: ToSocketAddrs>(
+    bind_addr: A,
+    server_certs: &Vec<Vec<u8>>,
+) -> Result<(Endpoint, Driver), Box<Error>> {
+    let client_cfg = configure_client(server_certs)?;
+    let mut endpoint_builder = Endpoint::new();
+    endpoint_builder.default_client_config(client_cfg);
+    let (endpoint, driver, _incoming) = endpoint_builder.bind(bind_addr)?;
+    Ok((endpoint, driver))
+}
+
+/// Configures server to listen on given address and port.
```

Comment:
This does not modify a configuration, but rather constructs an Endpoint with a particular sort of configuration.
```suggestion
/// Constructs a QUIC endpoint configured to listen for incoming connections on a certain address and port.
```